platform/app/public/config/default.js

```diff
@@ -149,6 +149,121 @@ window.config = {
         },
       },
     },
+    {
+      namespace: '@ohif/extension-default.dataSourcesModule.dicomweb',
+      sourceName: 'F4C',
```

Comment:
```suggestion
      sourceName: 'PROD_F4C',
```
Je renomme le centre en BDD